cmd/swarmctl/cluster/update.go

```diff
@@ -114,6 +119,11 @@ var (
 				if err != nil {
 					return err
 				}
+
+				if hbPeriod < time.Duration(0) {
```

Comment:
Just in Line 122:
```
spec.Dispatcher.HeartbeatPeriod = uint64(hbPeriod)
```
hbPeriod is signed, while after conversion, spec.Dispatcher.HeartbeatPeriod is always unsigned. So we should validate the duration in the client.